namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/probation-in-court-cases-queue.tf

```diff
@@ -0,0 +1,114 @@
+module "probation-in-court-cases-queue" {
```

Comment:
How about just `court-cases-queue` ?
We are in the court-porbation namespace already